js/index.js

```diff
@@ -0,0 +1,99 @@
+const btn = document.querySelector('#newbook');
+const bookForm = document.querySelector('#bookform');
+
+function Book() {
+  this.title = document.querySelector('#booktitle').value;
+  this.author = document.querySelector('#author').value;
+  this.read = false;
+}
+
+// get books from localstorage
+const getBooks = () => {
+  const lib = localStorage.getItem('library');
+  return lib ? JSON.parse(lib) : [];
+};
```

Comment:
- [ ] Let's reset form after successful submit. 
  You can use [`reset`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) on form.